en/docs/try-out/consent-manager.md

```diff
@@ -18,17 +18,6 @@ requirements and manages consents.
     http_method="GET,DELETE"
     ```
    
-2. Open the `<API_HOME>/repository/conf/deployment.toml` file and add the following gateway executor configurations for 
```

Comment:
unless specified in the developer docs, let's not remove these configs as well

Reply:
fixed in https://github.com/wso2/docs-ob-toolkit-cds/pull/19/commits/eeb820c41b402329079378586375b7a780a4a8aa